### PR TITLE
Make big numbers readable sepolia

### PR DESCRIPTION
**Pull request by kaleb-keny**

### CODE REVIEW

**Comment by kaleb-keny** [comment]
hey @noisekit i will push this, verify that no parameters were changed by comparing to mainnet then push mainnet, unfortunately the script says it's updating stuff, but nothing is being updated.
" Hmm yea, I think we'd want it to hash the steps after the template is compiled to avoid this, but I'd be concerned there would be unintended side-effects in that case." As per noah

File: omnibus-base-sepolia-andromeda.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name = "synthetix-omnibus"
-version = "20"
+version = "21"
```
Comment (noisekit) [inline omnibus-base-sepolia-andromeda.toml:2]: I don't believe we need to up this version as this change is a noop, purely cosmetics.
```suggestion
version = "20"
```

**Review by noisekit — APPROVED** [review]
minus the version bump - much better now, those bonk numbers omg

File: tomls/omnibus-base-sepolia-andromeda/perps/pepe.toml [hunk 2]
```diff
@@ -5,7 +5,7 @@ defaultValue = "0xd69731a2e74ac1ce884fc3890f7ee324b6deb66147055249568869ed700882
 defaultValue = "1200"
 
 [setting.perpsPepeSkewScale]
-defaultValue = "8400000000000"
+defaultValue = "<%= String(8_400_000_000_000) %>"
```
Comment (noisekit) [inline tomls/omnibus-base-sepolia-andromeda/perps/pepe.toml:8]: bonkers

Comment (kaleb-keny) [reply tomls/omnibus-base-sepolia-andromeda/perps/pepe.toml:8]: thank you for catching that one

**Pull request merged**